packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -211,129 +217,150 @@ export function getStylesConfig(wco: WebpackConfigOptions): webpack.Configuratio
-                    // Ex: /* autoprefixer grid: autoplace */
-                    // tslint:disable-next-line: max-line-length
-                    // See: https://github.com/webpack-contrib/sass-loader/blob/45ad0be17264ceada5f0b4fb87e9357abe85c4ff/src/getSassOptions.js#L68-L70
-                    outputStyle: 'expanded',
-                  },
-                },
-              },
-            ],
+    },
+  ];
+
+  const styleLanguages: {
+    extensions: string[];
+    mimetype?: string;
+    use: webpack.RuleSetUseItem[];
+  }[] = [
+    {
+      extensions: ['css'],
+      mimetype: 'text/css',
+      use: [],
+    },
+    {
+      extensions: ['sass', 'scss'],
+      use: [
+        {
+          loader: require.resolve('resolve-url-loader'),
+          options: {
+            sourceMap: cssSourceMap,
+          },
+        },
+        {
+          loader: require.resolve('sass-loader'),
+          options: {
+            implementation: sassImplementation,
+            sourceMap: true,
+            sassOptions: {
+              // bootstrap-sass requires a minimum precision of 8
+              precision: 8,
+              includePaths,
+              // Use expanded as otherwise sass will remove comments that are needed for autoprefixer
+              // Ex: /* autoprefixer grid: autoplace */
+              // tslint:disable-next-line: max-line-length
+              // See: https://github.com/webpack-contrib/sass-loader/blob/45ad0be17264ceada5f0b4fb87e9357abe85c4ff/src/getSassOptions.js#L68-L70
+              outputStyle: 'expanded',
+            },
           },
-          {
-            test: /\.less$/,
-            use: [
-              {
-                loader: require.resolve('less-loader'),
-                options: {
-                  implementation: require('less'),
-                  sourceMap: cssSourceMap,
-                  lessOptions: {
-                    javascriptEnabled: true,
-                    paths: includePaths,
-                  },
-                },
-              },
-            ],
+        },
+      ],
+    },
+    {
+      extensions: ['less'],
+      use: [
+        {
+          loader: require.resolve('less-loader'),
+          options: {
+            implementation: require('less'),
+            sourceMap: cssSourceMap,
+            lessOptions: {
+              javascriptEnabled: true,
+              paths: includePaths,
+            },
           },
-          {
-            test: /\.styl$/,
-            use: [
-              {
-                loader: require.resolve('stylus-loader'),
-                options: {
-                  sourceMap: cssSourceMap,
-                  stylusOptions: {
-                    compress: false,
-                    sourceMap: { comment: false },
-                    paths: includePaths,
-                  },
-                },
-              },
-            ],
+        },
+      ],
+    },
+    {
+      extensions: ['styl', 'stylus'],
```

Comment:
Cannot find any reference to `stylus` being a using as a file extension for stylus

```suggestion
      extensions: ['styl'],
```